is_all_unique_elements.js

```diff
@@ -0,0 +1,16 @@
+
+const isAllUniqueElements = (array) => {
+  let isAllUnique = true;
+
+  array.forEach((currentElement, currentIndex) => {
+    array.slice(currentIndex + 1).forEach((elementToCompare) => {
+      if (elementToCompare === currentElement) {
```

Comment:
The '===' operator could cause some issues if the elements are not the exact match (even in type) and this might create some issues, Using '==' operator would help in avoiding these issues.